### PR TITLE
feat: persons query modified to work with verification info

### PR DESCRIPTION
**Pull request by SergeyTatarenko**
It is assumed that the query will be used in new verification module at Admin panel

### CODE REVIEW

**Comment by MykhailoMe** [comment]
Cool, Sergey!
If I may, I would call like
  nhsVerificationStatus: PersonVerificationStatus
  nhsVerificationStatusComment: String

As we have comments in different entities, like nhs_comment...
image

File: __schema__/persons.graphql [hunk 1]
```diff
@@ -288,3 +330,39 @@ type ConfidantPerson {
...
+  "Status `VERIFICATION_NEEDED` for an unverified person."
+  VERIFICATION_NEEDED
+  "Status `IN_REVIEW` for an unverified person."
+  IN_REVIEW
+}
+
+"""
+List of person verification status reasons. According to PERSON_VERIFICATION_STATUS_REASONS dictionary
+"""
+enum PersonVerificationReason {
```
Comment (SergeyTatarenko) [inline __schema__/persons.graphql:361]: Update enum according to new status model

File: __schema__/persons.graphql [hunk 2]
```diff
@@ -288,3 +330,39 @@ type ConfidantPerson {
...
+  "Status `NOT_VERIFIED` for a person."
+  NOT_VERIFIED
+  "Status `VERIFIED` for a person."
+  VERIFIED
+}
+
+"""
+List of values of `PersonVerificationStatus` that represent persons who has no decision about verification yet.
+"""
+enum UnverifiedPersonStatus {
```
Comment (SergeyTatarenko) [inline __schema__/persons.graphql:351]: Update list of statuses that should be displayed in "Unverified person" list in Admin panel

File: __schema__/persons.graphql (hunk 1 above)
Comment (MykhailoMe) [reply __schema__/persons.graphql:361]: updated. Thank you!

File: __schema__/persons.graphql (hunk 2 above)
Comment (MykhailoMe) [reply __schema__/persons.graphql:351]: Updated! Thanks!

**Review by MykhailoMe — DISMISSED** [review]

**Review by MykhailoMe — APPROVED** [review]

**Pull request merged**